src/map/status.c

```diff
@@ -2217,7 +2217,7 @@ int status_calc_pc_(struct map_session_data* sd, enum e_status_calc_opt opt) {
 	}
 	bstatus->aspd_rate = 1000;
 	bstatus->ele_lv = 1;
-	bstatus->race = RC_DEMIHUMAN;
+	bstatus->race = RC_PLAYER;
```

Comment:
Considering that PCs are no longer Demi Humans now, we need to provide the same old behavior (DemiHuman+Player) to everything (especially items), at least in Pre-renewal.